### ISSUE TITLE
Unable to enable accessibility 

### ISSUE THREAD

**Issue by NoSkillITA** [Fate-Grand-Automata/FGA#1576]
### Preparation

- [X] I tested the [latest release](https://github.com/Fate-Grand-Automata/FGA/releases/tag/canary)
- [X] I looked at other issues (even the [closed ones](https://github.com/Fate-Grand-Automata/FGA/issues?q=is%3Aissue+is%3Aclosed))
- [X] I read the [Troubleshooting Guide](https://github.com/MathewSachin/Fate-Grand-Automata/wiki/Troubleshooting)

### FGO server

NA

### FGA build number

1951

### Describe the bug

![photo_5922571039043337086_y](https://user-images.githubusercontent.com/131621740/233929494-309fba72-71e8-4fbc-9736-34b482a7b1fb.jpg)

When i try to click accessibility FGA (CI)  says that for my security i can't enable this option

### Video

https://user-images.githubusercontent.com/131621740/233930937-a72255cd-9b13-4b91-b000-a489a4a0435f.mp4

### Device model

Xiamo POCO F3

### Android version

12

**Comment by vybze** [comment]
https://github.com/Fate-Grand-Automata/FGA/issues/1301

**Comment by ElBlindOne** [comment]
I had this problem because i just switched to a Poco M5 with MIUi 14 and found out a solution while trying stuff, I was enabling all the permissions possible when I noticed, in the App Info screen, right at the bottom there was an "Allow other restricted settings" or something similar, sorry I didn't pay much attention, but i enabled that one and it was possible to work as usual, also it's just gone from that screen, nowhere to be enabled again it seems, if anyone else sees it please screenshot it for others to see
